torch/_dynamo/eval_frame.py

```diff
@@ -563,7 +563,7 @@ def guard_export_print(guards):
 
 
 def export(
-    f, *args, aten_graph=False, decomposition_table=None, tracing_mode="real", **kwargs
+    f, *args, aten_graph=False, decomposition_table=None, tracing_mode="symbolic", **kwargs
```

Comment:
I am fine with this, but the default for make_fx is real, so we should make it clear that:

```
graph, guards = export(f, args, aten_graph=False)
make_fx(graph)(args)
```
will now not be the same as:
```
graph, guards = export(f, args, aten_graph=True)
```
because the defaults are no longer lined up.